README.md

```diff
@@ -1,19 +1,23 @@
 [![Build Status](https://travis-ci.org/ignitionrobotics/ros_ign.svg?branch=melodic)](https://travis-ci.org/ignitionrobotics/ros_ign/branches)
 
-* ROS 1 branches:
-    * [melodic](https://github.com/osrf/ros_ign/tree/melodic)
-        * Blueprint and Citadel
-        * Melodic
-    * [noetic](https://github.com/osrf/ros_ign/tree/noetic)
-        * Citadel
-        * Noetic
-* ROS 2 branches:
-    * [dashing](https://github.com/osrf/ros_ign/tree/dashing)
-        * Blueprint and Citadel
-        * Dashing and Eloquent
-    * [ros2](https://github.com/osrf/ros_ign/tree/ros2)
-        * Citadel
-        * Foxy
+ROS version | Ignition version | Branch | Binaries hosted at
+-- | -- | -- | --
+Melodic | Blueprint | [melodic](https://github.com/osrf/ros_ign/tree/melodic) | https://packages.osrfoundation.org
+Melodic | Citadel | [melodic](https://github.com/osrf/ros_ign/tree/melodic) | :x:
+Noetic | Blueprint | :x: | :x:
```

Comment:
If the branch does not support Blueprint in anyway we can remove it from the list or improve as  not supported somehow:

```suggestion
Noetic | Blueprint | not supported | :x:
```
same for Foxy below

Reply:
a84dba439f1ea13742f9b4785cf959c73c1d3c74